test/sqlsrv/srv_020_unicode_strings.phpt

```diff
@@ -0,0 +1,48 @@
+--TEST--
+Query non-ascii strings: sqlsrv_fetch_array
+--DESCRIPTION--
+Test sqlsrv_fetch_array() with non-ASCII values
+--SKIPIF--
+<?php require('skipif.inc'); ?>
+--FILE--
+<?php
+require_once("MsCommon.inc");
+
+// Connect
+$conn = Connect();
+if( !$conn ) { die( print_r( sqlsrv_errors(), true)); }
+
+// Create table
+$tableName = '#srv_020test';
+$stmt = sqlsrv_query($conn, "CREATE TABLE $tableName (FirstName VARCHAR(10), LastName NVARCHAR(20), Age INT)");
+if( $stmt === false ) { die( print_r( sqlsrv_errors(), true )); }
+sqlsrv_free_stmt( $stmt);
+
+// Insert data
+$sql = "INSERT INTO $tableName VALUES ('John', 'Doe', 30)";
+$stmt = sqlsrv_query( $conn, $sql);
+sqlsrv_free_stmt( $stmt);
+
+$sql = "INSERT INTO $tableName VALUES ('Nhoj', N'Eoduard', -3), ('Joe', N' I❤PHP', 2016)";
+$stmt = sqlsrv_query($conn, $sql);
+sqlsrv_free_stmt( $stmt);
```

Comment:
Yes it's possible. See this [doc](https://technet.microsoft.com/en-us/library/dd776381(v=sql.105).aspx)